draft-js-emoji-plugin/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## To be released
+
+- Replace legacy lifecycle hooks with UNSAFE aliases; the required react version is 16.3
```

Comment:
if we're dropping support for versions prior to 16.3 we need to bump the versions of the plugins since this is a breaking change, this'll be at least a major version change

Reply:
Yes, I mentioned about it in description
> This will require major bump in the next release. Though after fixing
storybook I will try to drop their usage so we could prevent major bump
for some packages.